guides/Running-Tests.md

```diff
@@ -0,0 +1,14 @@
+# Running Tests
```

Comment:
For my Temporal tests, I actually ran via the node script. Do you think it would be helpful to mention this route or do you think it's just a recipe for more misery?

Reply:
I'm actually not sure what node script you're referring to

Reply:
Maybe the `npm run test` thing? I'm sorry, no one should have to sit through running the entire test suite with a single thread pool :| The `test` script was originally used for CI, but then we made one called `ci` and `test` became a generic "run all the tests" command (https://github.com/tc39/test262/pull/2137/).

Reply:
I was referring to what seems to be an invocation of the CI script from [the Temporal Polyfill package.json](https://github.com/tc39/proposal-temporal/blob/main/polyfill/package.json#L14).

It seems to me that it would be desirable to have a script that can accept file glob arguments and run from the command line, as a low-key way to run specs one is working on and as a way to let writers, especially new ones, see run new or changed tests without shunting them into another project. I thought it already existed, but I was just lucky!

(I agree that not _only_ making it possible to run the entire suite is good, but if people want to do so, by not filtering it, who am I to stop them?)

Reply:
Ah, I see this is [covered below](https://github.com/tc39/test262/pull/3406#discussion_r809541417) — glad we all agree about running tests lo-key.